src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splitString = sourceString.split(';');
+  const dictionary = {};
+
+  splitString.forEach((element) => {
+    const [key, value] = element.split(':').map((item) => item && item.trim());
+
+    if ((key, value)) {
```

Comment:
The condition `(key, value)` is incorrect for checking if both `key` and `value` are present. This will always evaluate to the value of `value`, which may lead to unexpected behavior. Instead, use `if (key && value)` to ensure both `key` and `value` are truthy before adding them to the dictionary.